setup.cfg

```diff
@@ -19,14 +19,7 @@ classifiers =
     Operating System :: Microsoft :: Windows
     Operating System :: POSIX
     Programming Language :: Python :: 2
-    Programming Language :: Python :: 2.7
     Programming Language :: Python :: 3
-    Programming Language :: Python :: 3.5
-    Programming Language :: Python :: 3.6
-    Programming Language :: Python :: 3.7
-    Programming Language :: Python :: 3.8
-    Programming Language :: Python :: 3.9
-    Programming Language :: Python :: 3.10
     Programming Language :: Python :: Implementation :: CPython
```

Comment:
this is a new option in `setup-cfg-fmt` -- if we want these we can select `--include-version-classifiers` in `args`.  though I find them noisy so by default they are removed.  I don't have a preference for `tox` but I'm removing them in my projects

Reply:
I find these useful for checking compatibility with the next python version :thinking: not sure.